src/PythonBindings/setup.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from distutils.core import setup
+
+setup(
+    name='spectre',
+    version='0.1',
+    description="Python bindings for the SpECTRE code",
```

Comment:
`for the code SpECTRE` or just `for SpECTRE`

Reply:
Fixed this. I should note that we'll certainly want to extend the metadata in this file in the future, e.g. add a `long_description`, do proper versioning, add contact email addresses, and so on.

Reply:
Ah okay, cool! Once we get away from version 0 I guess we can set that according to the SpECTRE version?

Reply:
Yeah, we can do that with a CMake replace.